cmd/snap/cmd_info.go

```diff
@@ -503,6 +503,20 @@ func (iw *infoWriter) maybePrintNotes() {
 	return
 }
 
+func (iw *infoWriter) maybePrintCohortKey() {
+	if !iw.verbose {
```

Comment:
we should flag somehow that that a cohort is set even in non verbose mode, naive suggestion:

    cohort: true

but maybe it's confusing? suggestions?

Reply:
yeah `cohort: true` is silly imo, but i could add `in-cohort` to Notes, wdyt?